sawtooth-asset-app/src/main/java/com/mycompany/blockchain/sawtooth/app/config/Application.java

```diff
@@ -17,7 +17,7 @@
  *
  */
 @SpringBootApplication
-@ComponentScan("com.mycompany.blockchain.sawtooth.app")
+@ComponentScan("com.mycompany.blockchain.sawtooth")
```

Comment:
Put component scan at app level, other packages are base package classes and we will let applicaitons to define the beans. Please add .app.

Reply:
done